bower.json

```diff
@@ -46,6 +46,6 @@
   },
   "devDependencies": {
     "angular-mocks": "1.3.0 - 1.5.*",
-    "angular-ui-router": "^0.3.2"
+    "angular-ui-router": "1.0.0-beta.3"
```

Comment:
idk.  It is a dev dependency, which to me seems like it could be cutting edge since it shouldn't effect production code.  Just my 2 cents :-)